noetic/distribution.yaml

```diff
@@ -4657,11 +4658,15 @@ repositories:
       - nav_grid
       - nav_grid_iterators
       - nav_grid_pub_sub
+      - nav_grid_server
```

Comment:
@DLu I notice both the repo and this package is missing a LICENSE file, which [the review guidelines are now requiring](https://github.com/ros/rosdistro/blob/master/REVIEW_GUIDELINES.md#types-of-pull-requests). Mind adding one for the next release?